modules/core/src/main/java/org/apache/synapse/AbstractExtendedSynapseHandler.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Which license do we use here , should this be WSO2 or Apache , I'm also not pretty sure. @sdkottegoda @tishan89

Reply:
Unless there has been a recent change we should use WSO2 licence

Reply:
Changed to WSO2 Licence.